manifests/h/HIKARI-FIELD/HIKARI-FIELD-CLIENT/1.1.8/HIKARI-FIELD.HIKARI-FIELD-CLIENT.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: HIKARI-FIELD.HIKARI-FIELD-CLIENT
+PackageVersion: 1.1.8
+InstallerType: zip
+NestedInstallerType: nullsoft
+NestedInstallerFiles:
+- RelativeFilePath: HIKARI-FIELD-CLIENT-Setup-1.1.8.exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-03-07
+Installers:
+- Architecture: x86

```

Hi @xiSage, 

Our `x86` validations are performed on 32-bit OSes. Based on the error above, this appears to be an `x64` application.

```suggestion
- Architecture: x64
```